src/applications/vaos/reducers/newAppointment.js

```diff
@@ -345,6 +347,31 @@ export default function formReducer(state = initialState, action) {
         loadingFacilityDetails: true,
       };
     }
+    case FORM_SCHEDULE_APPOINTMENT_PAGE_OPENED: {
+      return {
+        ...state,
+        loadingAppointmentSlots: true,
```

Comment:
should this reducer also effectively reset the parts of the state that are set by the `FORM_SCHEDULE_APPOINTMENT_PAGE_OPENED_SUCCEEDED` reducer? so wiping out `availableDates` and `availableSlots` for example?